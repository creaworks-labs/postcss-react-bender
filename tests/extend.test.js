var { processFileAndMatchSnapshot } = require("./utils");

it("process @extend definition in styles", () => {
  return processFileAndMatchSnapshot("./tests/extend.test.scss");
});
