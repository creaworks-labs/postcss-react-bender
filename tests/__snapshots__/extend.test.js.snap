// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`process @extend definition in styles 1`] = `
Object {
  "developer.project.screen": Object {
    ".circle": Object {
      "alignItems": "center",
      "backgroundColor": "yellow",
      "justifyContent": "center",
    },
    "circle": Object {
      "backgroundColor": "yellow",
      "flexBasis": 0,
      "flexGrow": 1,
      "flexShrink": 1,
      "marginTop": 50,
      "overflow": "hidden",
    },
    "developer.project.view": Object {
      ".blue": Object {
        "backgroundColor": "blue",
      },
      ".nestedCircle": Object {
        "backgroundColor": "blue",
        "flexBasis": 0,
        "flexGrow": 0,
        "flexShrink": 1,
        "marginTop": 50,
        "overflow": "hidden",
      },
      ".square": Object {
        ".min": Object {
          "flexBasis": 0,
          "flexGrow": 0,
          "flexShrink": 1,
        },
        "alignItems": "center",
        "backgroundColor": "yellow",
        "justifyContent": "center",
      },
    },
  },
}
`;
